src/System.Net.Http/System.Net.Http.sln

```diff
@@ -0,0 +1,21 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.22816.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Net.Http.UnitTests", "tests\UnitTests\System.Net.Http.Unit.Tests.csproj", "{5F9C3C9F-652E-461E-B2D6-85D264F5A733}"
```

Comment:
I discussed this with @weshaggard.  As long as the test assembly ends in ".Tests" it is ok.  We also plan to have additional test assemblies such as "System.Net.Http.Tests".  That test assembly would be end-to-end testing and would link against the actual product System.Net.Http.dll itself.